src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -221,10 +221,31 @@ function ExportAnnotations({
             </Link>{' '}
             about copying and exporting annotations.
           </p>
+          <label htmlFor={userSelectId} className="block font-medium">
+            Select which user{"'"}s annotations to export:
+          </label>
+          <SelectNext
+            value={selectedUser}
+            onChange={setSelectedUser}
+            buttonId={userSelectId}
+            buttonContent={
+              <UserAnnotationsListItem userAnnotations={selectedUser} />
+            }
+            data-testid="user-select"
+          >
+            <SelectNext.Option value={allAnnotationsOption}>
+              <UserAnnotationsListItem userAnnotations={allAnnotationsOption} />
+            </SelectNext.Option>
+            {userList.map(userInfo => (
+              <SelectNext.Option key={userInfo.userid} value={userInfo}>
+                <UserAnnotationsListItem userAnnotations={userInfo} />
+              </SelectNext.Option>
+            ))}
+          </SelectNext>
           <label
             data-testid="export-count"
             htmlFor={fileInputId}
-            className="font-medium"
+            className="block font-medium"
```

Comment:
This is needed so that `space-y-3` in parent element is properly applied.

Reply:
I guess you could also use a vertical flex layout for the whole stack of components, which will wrap each element in an anonymous box as needed.

Reply:
This actually gives me an idea. We could wrap the two "block" ("select user" and "file name") into a flex-col container, and based on the feature flag, set it as reverse or not.

That way we don't need to change the KB articles yet, but there's also not a lot of duplicated content here.